scripts/apps/monitoring/directives/MonitoringGroup.ts

```diff
@@ -668,6 +668,8 @@ export function MonitoringGroup(
                             searchCriteria.source.size = PAGE_SIZE;
                         }
                     }
+                } else if (scope.group != null && scope.group.type === 'personal') {
+                    provider = 'production';
```

Comment:
Why doesn't archive work?

Reply:
it doesn't have published items

Reply:
how about `published` endpoint? It should have published items.

Reply:
it has only published, but not items in progress. so there is new api which has both

Reply:
Is this new API a part of the effort to unify the endpoints for articles?(like SDESK-4687) Does it have personal items? Is it read-only or supports post/put/patch as well?

Reply:
it is, for now it's read-only

Reply:
Does the querying work identically to `archive` / `published`? Can we safely replace  `archive` / `published` queries to `production`?

I'd also pick a better name for it like `articles` or `publishable`(so name includes media).

Reply:
it works like archive, I would do more testing before replacing it everywhere.
I think production is what we usually use when referring to this type of content, created by superdesk users and kind of live, so it's not ingest anymore and not archive yet

Reply:
What I don't like in the name is that it's closer to an adjective than a noun. It would be unclear to name a collection `saved`, because it's not clear **what** is saved. So we call it `saved_searches`.

The same goes for `published` or `archived`. What is published or archived? Articles. Calling it `articles_archived`, `articles_published` or `articles_production` is much clearer.

Reply:
yep saved/archived/published that's adjective, but production?

Reply:
`production` describes the state, not an entity. For example, it's possible to have production planning items, production events, production desk stages, production workspaces. It's not how it's currently structured, but if you aren't familiar with the codebase there is no way to know what is it referring to when the name of an entity is not included in the name.

Reply:
it's more about the process than state. adding articles to it is imo a bit misleading because there are pictures, packages, videos etc. `news_production` maybe, but what else would you produce in superdesk anyhow. for events that process is called planning. and as you said we don't use it anywhere else

Reply:
> what else would you produce in superdesk anyhow

some kind of templates, media, analytics reports. It sounds silly for us since we know that we don't have that, but for someone not too familiar with the system it's hard to tell.

I'd say it's a problem that we haven't come up with a name for the main entity in Superdesk yet. It seems we agree that `archived` or `published` or `saved` are bad names because they are specific to one entity and that entity name is not a part of a collection/endpoint name.

`IArticle` in the client is misleading for the same reason as you pointed out - it can contain a package or a media item too. It would be good to come up with a better name for that entity and use it everywhere. I like `news` / `news item` you suggested. Since `production` is the default stage, I'd rename `production` collection/endpoint to be simply `news` and have others be different like `news_archived` or `news_ingest`.

Reply:
:ok_hand: sounds like a plan